### PR TITLE
Update README with beta acknowledgement

### PR DESCRIPTION
**Pull request by addb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,23 @@
...
-## About this project
-
-The [`Media`](https://developer.adobe.com/client-sdks/documentation/adobe-media-analytics) extension represents the Adobe Experience Platform SDK's Media Analytics extension that provides clients with robust measurement for audio, video and advertisements.
-
-## Getting started
-
-Refer to the [getting started](./Documentation/getting-started.md) guide for setting up and using the SDK with your application.
+BETA ACKNOWLEDGEMENT
 
+AEPEdgeMedia is currently in Beta. Use of this code is by invitation only and not otherwise supported by Adobe. Please contact your Adobe Customer Success Manager to learn more.
```
Comment (emdobrin) [inline README.md:5]: The Media for Edge network extension is currently ..

File: README.md [hunk 2]
```diff
@@ -1,43 +1,23 @@
-# Adobe Experience Platform - Media extension for Android
+# Adobe Experience Platform Edge Media Mobile Extension - Android
```
Comment (emdobrin) [inline README.md:1]: for consistency with the other SDKs we can use `Adobe Experience Platform Media for Edge Network Extension`

File: README.md [hunk 3]
```diff
@@ -1,43 +1,23 @@
...
-| ------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------- |
-| [Core extensions](https://github.com/adobe/aepsdk-core-android)                      | The Mobile Core represents the foundation of the Adobe Experience Platform mobile SDK.               |
-| [Analytics extension](https://github.com/adobe/aepsdk-analytics-android)             | The Analytics extension allows for sending mobile interaction data from your app to Adobe Analytics. |
-| [Assurance extension](https://github.com/adobe/aepsdk-assurance-android)             | The Assurance extension enables validation workflows for your SDK implementation.                    |
-| [AEP SDK sample app for Android](https://github.com/adobe/aepsdk-sample-app-android) | Contains Android sample app for the AEP SDKs.                                                        |
+The Media extension represents the Adobe Experience Platform SDK's Edge Media Analytics extension that provides clients with robust measurement for audio, video and advertisements using Edge Network.
```
Comment (emdobrin) [inline README.md:11]: we can skip about for now

File: README.md (hunk 2 above)
Comment (addb) [reply README.md:1]: I checked the edgeIdentity and edgeConsent and followed this

**Review by emdobrin — APPROVED** [review]

**Pull request merged**